cli_scripts/voice_coding_cli.py

```diff
@@ -0,0 +1,26 @@
+import subprocess,sys,click
+
+@click.command()
+@click.option('--voice/--no-voice', default = False, help = "Use --voice if you want to code using your voice.")
+@click.option('--speech', prompt = "What is your default Speech Recognition engine ? ", help = "Choose your default Speech Recognition engine")
+def enable_voice_coding(voice,speech):
+    """Welcome to LarynxCode. This is a simple CLI that enables you to allow voice programming on your editor."""
+    if voice:
```

Comment:
I don't think the `voice` option/variable is needed, it just adds complexity to the code

Reply:
Actually, we'll be adding another option, --caster-help. Haven't pushed it yet because it needs the web app to be ready.